Cataclysm/Rogue.lua

```diff
@@ -1345,8 +1442,9 @@ spec:RegisterAbilities( {
         gcd = "totem",
 
         spend = function()
-            if talent.improved_sinister_strike.rank == 2 then return 40 end
-            if talent.improved_sinister_strike.rank == 1 then return 42 end
+            if talent.improved_sinister_strike.rank == 2 then return 39 end
```

Comment:
rank == 3?

This block could be replaced with `return 45 - ( talent.improved_sinister_strike.rank * 2 )`, no?

Reply:
for cata its 3 se now i missed rank3 but ya think it can be changed to return 42 - ( talent.improved_sinister_strike.rank * 3 )